Makefile

```diff
@@ -67,17 +64,20 @@ else
 endif
 
 test-with-kind: fixture
-	@build/docker/build-image.sh intel/intel-fpga-admissionwebhook buildah
-	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:devel docker.io/intel/intel-fpga-admissionwebhook:devel
-	@mkdir -p $(e2e_tmp_dir)
-	@$(PODMAN) save "docker.io/intel/intel-fpga-admissionwebhook:devel" -o $(e2e_tmp_dir)/$(WEBHOOK_IMAGE_FILE)
-	@$(KIND) create cluster --name "intel-device-plugins" --kubeconfig $(e2e_tmp_dir)/kubeconfig --image "kindest/node:v1.19.0"
-	@$(KIND) load image-archive --name "intel-device-plugins" $(e2e_tmp_dir)/$(WEBHOOK_IMAGE_FILE)
-	$(KUBECTL) --kubeconfig=$(e2e_tmp_dir)/kubeconfig apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
-	@$(GO) test -v ./test/e2e -args -kubeconfig $(e2e_tmp_dir)/kubeconfig -kubectl-path $(KUBECTL) -ginkgo.focus "FPGA Admission" || rc=1; \
-	$(KIND) delete cluster --name "intel-device-plugins"; \
-	rm -rf $(e2e_tmp_dir); \
-	exit $$rc
+	# Build Images
+	make intel-sgx-admissionwebhook
+	make intel-fpga-admissionwebhook
+	make intel-deviceplugin-operator
+	# Build a Cluster with KinD & Install Cert-Manager 
+	go get sigs.k8s.io/kind@${KIND_VERSION}
```

Comment:
I'm thinking we could we have some `install-tools`  target where we put all these. See also #623

Reply:
Got it, I assigned #623 to myself! :)